source/math/FloatSize3D.ooc

```diff
@@ -79,6 +79,9 @@ FloatSize3D: cover {
 		array := input split(',')
 		This new (array[0] toFloat(), array[1] toFloat(), array[2] toFloat())
 	}
+	linearInterpolation: static func (a, b: This, ratio: Float) -> This {
+		This new(Float linearInterpolation(a width, b width, ratio), Float linearInterpolation(a height, b height, ratio), Float linearInterpolation(a depth, b depth, ratio))
+	}
```

Comment:
Once every PR is in, we could try to crate naming and argument consistency with color's Blend function. Both could be called `mix` as GLSL does for both colors and vectors.